showcase/app/templates/components/table.hbs

```diff
@@ -949,7 +949,7 @@
         <B.Td><Hds::Link::Inline @href="#showcase">{{B.data.artist}}</Hds::Link::Inline></B.Td>
         <B.Td>
           <div class="shw-component-table-cell-content-div">
-            <FlightIcon @name={{B.data.icon}} />
+            <Hds::Icon @name={{B.data.icon}} />
```

Comment:
Looking at Percy, this instance of changing from FlightIcon to Hds::Icon broke the spacing.. Is this expected?

<img width="1176" alt="Screenshot 2024-08-15 at 10 14 00 AM" src="https://github.com/user-attachments/assets/06539eed-774f-47f0-af31-57ca24a01880">

Reply:
Yes, there will be some instances where there are visual regressions that we need to fix manually. We switched from defaulting to `block` display rather than defaulting to `inline-block`.